### PR TITLE
Filters

### PR DESCRIPTION
**Pull request by syed-shaida-hussain**
Added filter by price and category and added images to products.
[Live link ](https://playful-kleicha-f713f0.netlify.app/)

### CODE REVIEW

File: src/components/Filters.js [hunk 1]
```diff
@@ -2,63 +2,125 @@ import {useFilters } from "../contexts/filter-context/filter-context"
...
+            </div>
+            <div className="rating">
+                <input id="rating2" className="margin-left" name="ratings" type="radio" onChange={() =>
+            dispatch({ type: "RATE", payload: "3_STARS_AND_ABOVE" })
+            }
+            checked={rateBy && rateBy === "3_STARS_AND_ABOVE"}  />
+                <label for="rating2"> 3 stars and above</label>
+            </div>
+            <div className="rating">
+                <input id="rating3" className="margin-left" name="ratings" type="radio" onChange={() =>
```
Comment (ranitasahaa1230) [inline src/components/Filters.js:62]: For rating, use array. It would reduce your lines of code too

**Review by ranitasahaa1230 — APPROVED** [review]
- Check the category COD available, in my case its not working
- Add to wishlist is not happening from the product page
- After adding items to cart, the 'Add to cart' button should show 'go to cart'
- From the wishlist page, move to cart is not happening
- You can remove the search bar from the home page

Rest looks good, Great UI

File: src/components/Filters.js (hunk 1 above)
Comment (syed-shaida-hussain) [reply src/components/Filters.js:62]: For me , COD available is working fine.
Fixed Add to wishlist from product page.
After adding items to cart, the 'Add to cart' button increases only quantity of the product.
From the wishlist page, move to cart is only moving it to cart but not deleting from wishlist. Check the quantity of cart items after clicking on move to cart button.

**Review by tushargoyal812 — COMMENTED** [review]
looks good to merge.

**Pull request merged**